README.md

```diff
@@ -15,55 +26,31 @@ In your Docker engine you'll now see:
 
 You can now run `scripts/run.sh` to re-run the testing suite, using whatever arguments you like. To see the full list of supported arguments, pass in the `--help` flag to the `run.sh` script.
 
-## Developing Locally
-Some info:
-* The initializer binary is produced in `build/ava-e2e-tests` every time `scripts/rebuild_initializer_binary.sh` is run; you can run this binary with the `--help` flag to see detailed start options
-* The `run.sh` script is a convenience for calling the binary with sane default arguments (**NOTE:** you can pass in a comma-separated list of test names as the argument to run just those tests)
-* The controller Docker image is produced to the Docker image tag `kurtosistech/ava-e2e-tests_controller:latest` (at time of writing) every time `scripts/rebuild_controller_image.sh` is run
-* Every time you change:
-    1. The initializer you'll need to rerun `scripts/rebuild_initializer_binary.sh`
-    2. The controller you'll need to rerun `scripts/rebuild_controller_image.sh`
-    3. The commons code you'll need to rerun both
-
-### Helpful Tip
-You can use the following alias to stop & remove Docker containers of a certain type:
+Developing Locally
+------------------
+### Architecture
+This repo uses the [Kurtosis architecture](https://github.com/kurtosis-tech/kurtosis), so you should first go through the tutorial there to familiarize yourself with the core Kurtosis concepts.
```

Comment:
how are we going to handle this if we make the repo private / closed source?

Reply:
I think that's part of the "how do we give clients of Kurtosis the documentation". The Ava implementation is a client, so they'd have access to the documentation (however it's distributed)